app/src/main/res/xml/preferences_user_interface.xml

```diff
@@ -21,6 +21,13 @@
                 android:summary="@string/pref_episode_cover_summary"
                 android:defaultValue="true"
                 android:enabled="true"/>
+        <SwitchPreferenceCompat
+            android:title="@string/pref_show_remain_time_title"
```

Comment:
Please use the same indention as the rest of the file

Reply:
The file is actually inconsistent.  I've auto-idented

Reply:
Oh, sorry. In that case I would prefer to not re-format until we have a fixed code style with CI checks.

Reply:
np, ready for a review @ByteHamster